### ISSUE TITLE
adding defaultSitespeedioConfigFile file is giving error as Found. Redirecting to

### ISSUE THREAD

**Issue by Ankit3794** [sitespeedio/onlinetest#74]

**Comment by soulgalore** [comment]
Hi @Ankit3794 thanks for creating the issue. So it's three different issues right? 
1. One is that it seems like configuration on the server isn't propagated to the the actual test (the link for the header)
2. The result isn't redirected
3. The result page in search isn't linked.

What OS do you use and how do you run (do you use Docker or NodeJS)?

**Comment by Ankit3794** [comment]
Hi @soulgalore, 

I am running on Ubuntu 22.04 directly with Server and Test Runner both are running directly with NodeJS, where as TestRunner will be starting Docker Container to run actual test.

Yes correct, kind of three issues
1. defaultConfig given in server/default.yaml is not being carry forwarded
2. Result is not being redirected
3. Result page in search is not linked.

But all these issues are only happening if defaultConfig is given in server/default.yaml file. If defaultConfig is not given 2nd and 3rd issue is not happening.

**Comment by soulgalore** [comment]
That is strange, I have a setup where I run on Ubuntu where all the three things work, however they run on two separate servers so I don't use localhost. And you don't get any errors in your logs? I'll check later this weekend and see if I understand what's going on.

**Comment by Ankit3794** [comment]
@soulgalore , Debugged the issue over the weekend and seen that once the add-test is done on `result.get('/:id')` navigation job.status is completed but `job.returnvalue.pageSummaryUrl` is empty

https://github.com/sitespeedio/onlinetest/blob/822e97a3887f2640263c0b6b1851b722c682e415/server/src/routes/html/result.js#L20

**Comment by soulgalore** [comment]
Thanks for looking into it. In the logs of the testrunner, do you get a log that says something like _Find the result at_  followed by a URL? In your sitespeed.json file do you have setup for the result URL, something like:
```
{
 "resultBaseURL": "http://127.0.0.1:9000/sitespeedio"
}
```

**Comment by soulgalore** [comment]
> defaultConfig given in server/default.yaml is not being carry forwarded

I'm working on this, hope to have some clue about it tomorrow.

**Comment by soulgalore** [comment]
Hmm, for me setting the default config both on the server and testrunner like this works:

```
{
  "html": {
    "homeurl": "http://localhost:3000"
  }
}
```

However setting some Chrome args on the test runners default file fails for me, so there's some bug/unexpected behaviour going on here.

**Comment by Ankit3794** [comment]
> Thanks for looking into it. In the logs of the testrunner, do you get a log that says something like _Find the result at_ followed by a URL? In your sitespeed.json file do you have setup for the result URL, something like:
> 
> ```
> {
>  "resultBaseURL": "http://127.0.0.1:9000/sitespeedio"
> }
> ```

Yes, in sitespeed.json file I have resultBaseURL setup available. 

I am not seeing any logs on testrunner stating Find the result at

![image](https://github.com/user-attachments/assets/17ebd742-09fc-44a5-80c7-1229dc98904c)
